xml/System.IO/File.xml

```diff
@@ -150,10 +150,32 @@
         <Parameter Name="bytes" Type="System.Byte[]" Index="1" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="path">To be added.</param>
-        <param name="bytes">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="path">The file to append to.</param>
+        <param name="bytes">The bytes to append to the file.</param>
+        <summary>
+          <para>Appends the specified byte array to the end of the file at the given path.</para>
+          <para>If the file doesn't exist, this method creates a new file.</para>
+        </summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="path" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="bytes" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="path" /> is empty.</exception>
+        <exception cref="T:System.IO.PathTooLongException">The specified path, file name, or both exceed the system-defined maximum length.</exception>
+        <exception cref="T:System.IO.DirectoryNotFoundException">The specified path is invalid (for example, it is on an unmapped drive).</exception>
+        <exception cref="T:System.IO.IOException">An I/O error occurred while opening the file.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">
+          <paramref name="path" /> specified a file that is read-only.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">
+          <paramref name="path" /> specified a file that is hidden.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">
+          <paramref name="path" /> specified a directory.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">The caller does not have the required permission.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">This operation is not supported on the current platform.</exception>
```

Comment:
Shouldn't this be merged into one `<exception cref="T:System.UnauthorizedAccessException">` with `- or -`s between each reason?
![image](https://github.com/user-attachments/assets/fed87d8f-57f9-452f-bec7-5b610fa382cc)

Reply:
Yeah, the tool was supposed to group them. Can you help me with suggestions?

Reply:
But this means they were added as individual separate exceptions in triple slash comments instead of placed in a single exception.